data/json/items/tool_armor.json

```diff
@@ -178,6 +178,25 @@
     "use_action": "PORTABLE_GAME",
     "magazines": [ [ "battery", [ "light_minus_battery_cell", "light_minus_atomic_battery_cell", "light_minus_disposable_cell" ] ] ]
   },
+  {
+    "type": "TOOL_ARMOR",
+    "id": "fitness_band",
+    "name": "fitness band",
+    "name_plural": "fitness bands",
```

Comment:
Probably unnecessary to specify plural name if the correct one is formed by adding "s" to name end (that's what is done if plural name is unspecified).